docs/admin/deployment/index.md

```diff
@@ -17,9 +18,9 @@ Use the following Admin guides to set up NeMo Curator in a production environmen
 
 Before deploying NeMo Curator in a production environment, review the comprehensive requirements:
 
-- **System**: Ubuntu 22.04/20.04, Python 3.10+
+- **System**: Ubuntu 22.04/20.04, Python 3.10 or 3.12 (Python 3.11 not supported)
```

Comment:
Not sure where this knowledge comes from, I know it is all over our repository for some reason. I don't think it's true that Python 3.11 is not supported, but maybe someone else can correct me?